### PR TITLE
Bump up dependencies

### PR DESCRIPTION
**Pull request by david98**
This should fix issue #1.

### CODE REVIEW

**Comment by zhaluza** [comment]
Appreciate it @david98 ! I'll take a look at your changes.

**Comment by david98** [comment]
I just ran a quick `yarn upgrade` and then all tests passed. Plus, I forced install the published version and it works just fine with react 17 so I think it really was only a matter of removing the inconvenience of `--force` 😀

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "test": "jest --verbose",
     "lint": "prettier --check .",
     "format": "prettier --write .",
-    "postinstall": "husky install",
+    "_postinstall": "husky install",
```
Comment (zhaluza) [inline package.json:16]: Any reason for the added underscore here?

Comment (zhaluza) [reply package.json:16]: Ah, I see that this is something that Husky does automatically (noticed the same thing in my local version too)

**Review by zhaluza — APPROVED** [review]

**Pull request merged**